tests/test_pipeline/test_pipeline.py

```diff
@@ -23,11 +25,52 @@
 from etna.pipeline import Pipeline
 from etna.transforms import AddConstTransform
 from etna.transforms import DateFlagsTransform
+from etna.transforms import FilterFeaturesTransform
+from etna.transforms import LogTransform
 from tests.utils import DummyMetric
 
 DEFAULT_METRICS = [MAE(mode=MetricAggregationMode.per_segment)]
 
 
+@pytest.fixture
+def sinusoid_ts():
+    periods = 100
+    sinusoid_ts_1 = pd.DataFrame(
+        {
+            "segment": 0,
+            "timestamp": pd.date_range(start="1/1/2018", periods=periods),
+            "target": [np.sin(i / 10) + i / 5 for i in range(periods)],
+            "feature_1": [i / 10 for i in range(periods)],
```

Comment:
You don't actually need so much features here, just leave 1-2(in your test you only use one)